helm/reana/templates/uwsgi-config.yaml

```diff
@@ -25,9 +25,11 @@ data:
                           # https://uwsgi-docs.readthedocs.io/en/latest/articles/SerializingAccept.html
 
     # Logging
+    {{- if .Values.components.reana_server.uwsgi.log_http_errors_only }}
```

Comment:
One disadvantage of having only one true/false helm value representing both 4xx and 5xx is that some admins wouldn't be easily able to choose only 4xx or only 5xx if they wish so. Moreover, considering that there are more `log-*` options that admins might be interested to log, such as `log-ioerror`, perhaps we could make it a list?  

However I'm not sure it is worth the trouble, because somebody else might wish to `log-slow` which would need an argument... It would not be advantageous to do any complex argument handling here in Helm for those cases. (But rather to offer a way to pass the full textual snippet.) 

Just musing out loud whether one helm value is enough to represent "logging errors only", or whether we introduce more helm values to have more granularity (`reana_server.uwsgi.log_all`, `reana_server.uwsgi.log_4xx`, `reana_server.uwsgi.log_5xxx`).

Reply:
I had this very same doubt and I decided to go for a simpler approach, but indeed it's not very flexible... Maybe we can just replicate the very same uwsgi values?

However.. I'm just thinking that `uwsgi-config` is a config map so admins should be able to upgrade it? Although, I'm afraid that changes will be lost after an upgrade..

Reply:
Added the following configuration options and default values to make it more granular:
- `log_all: true`
- `log_4xx: true`
- `log_5xx: true`